src/rpc_client/tests.rs

```diff
@@ -215,209 +218,193 @@ mod multi_call_results {
         }
     }
 
-    mod reduce_with_stable_majority_by_key {
```

Comment:
The tests in that module were completely deleted (since the method `reduce_with_stable_majority_by_key` was removed)